core/src/main/java/org/apache/accumulo/core/cli/ClientOpts.java

```diff
@@ -195,19 +129,15 @@ public AuthenticationToken getToken() {
   @Parameter(names = {"-fake", "--mock"}, description = "Use a mock Instance")
   public boolean mock = false;
 
-  @Parameter(names = "--site-file",
-      description = "Read the given accumulo site file to find the accumulo instance")
-  public String siteFile = null;
-
   @Parameter(names = "--ssl", description = "Connect to accumulo over SSL")
   public boolean sslEnabled = false;
 
   @Parameter(names = "--sasl", description = "Connecto to Accumulo using SASL (supports Kerberos)")
   public boolean saslEnabled = false;
 
   @Parameter(names = "--config-file", description = "Read the given client config file. "
-      + "If omitted, the path searched can be specified with $ACCUMULO_CLIENT_CONF_PATH, which "
-      + "defaults to ~/.accumulo/config:$ACCUMULO_CONF_DIR/client.conf:/etc/accumulo/client.conf")
+      + "If omitted, the following paths will be searched ~/.accumulo/accumulo-client.properties:"
+      + "$ACCUMULO_CONF_DIR/accumulo-client.properties:/etc/accumulo/accumulo-client.properties")
```

Comment:
I'm not a fan of the file names ending in `.properties`. It's longer to type, and it exposes implementation detail a bit. I think I prefer the old `client.conf` file name, or maybe `accumulo-client.conf`. Changing conventions like this can be a pain for sysadmins... although client configuration was probably never used much because it wasn't a very mature feature before, so maybe it's okay to change the convention. Still, I'm not a fan of the `.properties` extension.

Reply:
In general, I like having accumulo in the name as the file can be used in a user's project and its clear what it is for. I like properties in the name as it's more descriptive but I would be OK with `accumulo-client.conf`. However, this should another issue as more code (outside of this PR) needs to changed if we start using another name.